bsp/esp_bsp_generic/include/bsp/esp_bsp_generic.h

```diff
@@ -0,0 +1,277 @@
+/*
+ * SPDX-FileCopyrightText: 2023 Espressif Systems (Shanghai) CO LTD
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * @file
+ * @brief ESP BSP: Generic
+ */
+
+#pragma once
+
+#include "sdkconfig.h"
+#include "driver/gpio.h"
+#include "driver/i2c.h"
+#include "driver/sdmmc_host.h"
+#include "iot_button.h"
+#include "led_indicator.h"
+
+/**************************************************************************************************
+ *  Pinout
+ **************************************************************************************************/
+/* I2C */
+#define BSP_I2C_SCL      (CONFIG_BSP_I2C_GPIO_SCL)
+#define BSP_I2C_SDA      (CONFIG_BSP_I2C_GPIO_SDA)
+
+/* USB */
+#if ESP_ROM_USB_OTG_NUM > 0
+#define BSP_USB_POS      (USBPHY_DP_NUM)
```

Comment:
`USBPHY_DP_NUM` and `USBPHY_DM_NUM` are defined in `usb_pins.h` which is not included in this file.

I suggest removing the USB section for now